experimental/QuadFormAndIsom/test/setup_tests.jl

```diff
@@ -1 +0,0 @@
-../../../test/Serialization/setup_tests.jl
```

Comment:
This file is actually a symbolic to `../../../test/Serialization/setup_tests.jl`. I assume it is a copy now because you used `sed`. (At least that's what happened to me in the past.)